src/polytopes/segment.jl

```diff
@@ -33,3 +33,20 @@ function (s::Segment)(t)
   a, b = s.vertices
   a + t * (b - a)
 end
+
+function Base.in(p::Point{2}, s::Segment{2})
+  # get vertices and coordinates
+  a, b = s.vertices
+  p1, p2, p3 = coordinates.((a, b, p))
+  # (i)  collinearity
+  arecollinear = (p2[2] - p1[2]) * (p3[1] - p2[1]) - (p3[2] - p2[2]) *
```

Comment:
Can you leave a comment about this expression? This is the cross-product of ...?

Reply:
Done! There are a few ways to get to this expression (slopes, cross-product, determinant), but know I modified it to make clear that you can obtain it from a cross-product (assuming one dimension is all equals to zero).